backend/src/types.ts

```diff
@@ -466,36 +466,23 @@ export type ODHSegmentKey = {
 
 export type BYONImage = {
   id: string;
-  user?: string;
-  uploaded?: Date;
-  error?: string;
-} & BYONImageCreateRequest &
-  BYONImageUpdateRequest;
-
-export type BYONImageCreateRequest = {
+  provider: string;
+  imported_time: string;
+  error: string;
   name: string;
   url: string;
-  description?: string;
-  // FIXME: This shouldn't be a user defined value consumed from the request payload but should be a controlled value from an authentication middleware.
```

Comment:
Is this taken care of already?

Reply:
Not sure if we should be removing this comment then. If we are, we should be tracking it somewhere. Are we already tracking this?

Reply:
No, i think we should just keep the comment here until we get to refactor the backend in phase 2.

Reply:
please keep the comment then, thanks